docs/index.md

```diff
@@ -8,7 +8,7 @@ import VPButton from "vitepress/dist/client/theme-default/components/VPButton.vu
 
 # 👨‍⚖️ `jstz`
 
-`jstz` (pronouced 'justice') is a JavaScript server runtime for Tezos 2.0
+`jstz` is a JavaScript server runtime powered by Tezos Smart Optimistic Rollups
```

Comment:
As with the other PRs that got merged recently, I think the removal of the pronunciation warrants a larger discussion. 

Here are my primary arguments _for_ a defined pronunciation: **Clarity and consistency**; `jstz` is an invented name, this could lead to confusion if there are multiple interpretations of how to pronounce the name. Establishing a defined pronunciation ensures clarity and consistency in any communications we have

There are multiple scenarios I could think of where this could cause confusion:
- Marketing / Advertisement of `jstz`
- Stakeholder engagement/service
- IP: Trademarks / IP rights, having a consistent product name (both capitalisation and pronunciation) is helpful (I think™️)